### ISSUE TITLE
Fix token selection in web view

### ISSUE THREAD

**Issue by TimJentzsch** [not-an-aardvark/reddit-oauth-helper#7]
You currently can't select the generated tokens in the web view, making it hard to copy them to the clipboard.
This seems to be due to the `disabled` property on the `input` elements, this should be replaced by the `readonly` property.

**Comment by not-an-aardvark** [comment]
I can select the tokens in Chrome (maybe it's browser-specific), but if you're interested in submitting a PR to fix this I'd be happy to take a look.

**Comment by TimJentzsch** [comment]
You are right, this seems to be specific to Firefox. I will still try to fix it.
